stock_packaging_calculator/models/product.py

```diff
@@ -100,7 +100,12 @@ def _product_qty_by_packaging(self, pkg_by_qty, qty, with_contained=False):
         for pkg in pkg_by_qty:
             qty_per_pkg, qty = self._qty_by_pkg(pkg.qty, qty)
             if qty_per_pkg:
-                value = {"id": pkg.id, "qty": qty_per_pkg, "name": pkg.name}
+                value = {
```

Comment:
Would you mind adding a hook here? I think it can be helpful to easily map the information you need without needing to browse the record using the ID downstream.

Reply:
Done here as I also needed it: https://github.com/Tonow-c2c/stock-logistics-warehouse/pull/1 :wink:

Reply:
Thanks!